packages/components-angular/src/app/pages/overview.component.ts

```diff
@@ -403,6 +411,14 @@ import { ChangeDetectionStrategy, Component } from '@angular/core';
         </my-prefix-p-select-wrapper>
       </div>
 
+      <div class="playground light" title="should render default multi-select with custom prefix">
+        <my-prefix-p-multi-select p-multi-select [name]="'options'" [label]="'Some label'">
+          <p-multi-select-option [value]="'a'">Option A</p-multi-select-option>
```

Comment:
Prefixes are missing here or to be more precise in html file that is the input to generate this.